cmd/cli/app/auth/offline_token/offline_use.go

```diff
@@ -57,8 +57,9 @@ func offlineUseCommand(_ context.Context, cmd *cobra.Command, _ []string, _ *grp
 
 	issuerUrlStr := clientConfig.Identity.CLI.IssuerUrl
 	clientID := clientConfig.Identity.CLI.ClientId
+	realm := clientConfig.Identity.CLI.Realm
 
-	creds, err := cli.RefreshCredentials(tok, issuerUrlStr, clientID)
+	creds, err := cli.RefreshCredentials(tok, issuerUrlStr, realm, clientID)
```

Comment:
Is it possible for us to not require a realm here? would be ideal to move away from that.

Reply:
Agree that it would be ideal.  I may try to hack that together this weekend.